repondeur/zam_repondeur/fetch/an/amendements.py

```diff
@@ -296,8 +294,8 @@ def _create_or_update_amendement(
         groupe = get_groupe(raw_auteur, amendement.num)
         auteur = get_auteur(raw_auteur)
 
-    dispositif = unjustify(get_str_or_none(amend, "dispositif") or "")
-    objet = unjustify(get_str_or_none(amend, "exposeSommaire") or "")
+    expose = unjustify(get_str_or_none(amend, "dispositif") or "")
+    corps = unjustify(get_str_or_none(amend, "exposeSommaire") or "")
```

Comment:
I'm quite surprised that my chain of renaming leads to putting the `exposeSommaire` into the `corps` and not the `expose`.  There is another place where it was quite weird too, let me reference it too.